.pre-commit-config.yaml

```diff
@@ -18,8 +18,8 @@ repos:
     rev: 19.10b0
     hooks:
       - id: black
-  - repo: https://github.com/prettier/pre-commit
-    rev: "v1.19.1"
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: "v2.1.2"
```

Comment:
This was to get the pre-commit hooks working again locally

Reply:
reference: https://github.com/OCA/oca-addons-repo-template/pull/41

Reply:
You can see here that prettier was working fine with the old config: https://travis-ci.com/github/OCA/sale-workflow/jobs/443358674#L284-L369

Could you please leave it as it was?